source/graphql/migrate-wundergraph.txt

```diff
@@ -0,0 +1,197 @@
+.. _migrate-wundergraph:
+
+==============================
+Migrate GraphQL to WunderGraph
+==============================
+
+.. meta::
+  :description: Learn how to migrate your GraphQL host from Atlas App Services to WunderGraph.
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. important::
+
+   Always refer to the official documentation of both MongoDB Atlas and 
+   `WunderGraph <https://docs.wundergraph.com/docs/supported-data-sources/mongodb-atlas>`__
+   for the most up-to-date and accurate information. Specific steps may vary
+   depending on the details of your project and the technologies used.
+
+`WunderGraph <https://wundergraph.com/>`__ is a comprehensive developer platform
+that comes equipped with an extensive variety of open-source tools and Software
+Development Kits (SDKs).
+
+One of these tools is a `Backend For Frontend (BFF) Framework 
+<https://docs.wundergraph.com/docs/use-cases/backend-for-frontend>`__. This BFF
+Framework is a robust, specification-compliant server framework that's you can 
+add to production environments. This framework can integrate data from any
+source. This includes data from MongoDB Atlas. Also, if you plan on using
+GraphQL Federation, take a look at `WunderGraph Cosmo 
+<https://cosmo-docs.wundergraph.com/>`__.
+
+Migrate to WunderGraph
```

Comment:
I'd tweak this intro slightly:

The WunderGraph BFF is an open source gateway that bundles up your data and generates type-safe clients for your frontend framework. 
You can also use hooks to add additional business logic, or 
[custom GraphQL resolvers](https://docs.wundergraph.com/docs/features/custom-graphql-resolvers) to decouple the clients from the database.

The following is an overview of how to migrate your GraphQL host from Atlas App Services to WunderGraph. Specific steps may vary depending on the details of your project and the technologies used.

- If you are starting from scratch, follow the 
[MongoDB / Atlas Data Source documentation](https://docs.wundergraph.com/docs/wundergraph-config-ts-reference/configure-mongodb-atlas-data-source). 
- If you plan on using GraphQL Federation, take a look at `WunderGraph Cosmo <https://cosmo-docs.wundergraph.com/>`__.

Reply:
Thanks. This structure makes a lot more sense. 😄